appium/webdriver/webdriver.py

```diff
@@ -585,6 +603,6 @@ def _addCommands(self):
         self.command_executor._commands[Command.LOCATION_IN_VIEW] = \
             ('GET', '/session/$sessionId/element/$id/location_in_view')
         self.command_executor._commands[Command.GET_DEVICE_TIME] = \
-            ('GET', '/session/$sessionId/appium/device/system_time')
```

Comment:
@KazuCocoa 

Can I change to only 'POST'?
My motivation is just to avoid complicated naming around below codes.

https://github.com/appium/python-client/blob/a40a933a906fe4cc8d31c83d55ba52f4830242b3/appium/webdriver/webdriver.py#L587-L588

e.g.
```
 self.command_executor._commands[Command.GET_DEVICE_TIME_GET] = \ 
('GET', '/session/$sessionId/appium/device/system_time') 
 self.command_executor._commands[Command.GET_DEVICE_TIME_POST] = \ 
('POST', '/session/$sessionId/appium/device/system_time') 
```
Your idea will be different from other variable naming..

If GET behavior should be kept, I'll update so.
(I'm not sure the impact to current behavior  by change from POST to GET.)

Thanks.

Reply:
You should define another command like your _e.g._ as compatibility for released Appium servers.
Current released appium server cannot handle the post method. We observe users who use older Appium servers. Other libraries also use this library in their libs.

Thus, we would like to avoid breaking changes as possible.
(Instead, please leave a comment we can remove it in the future, for instance.)

Reply:
I'd also keep the old GET endpoint. I was thinking on that and I think it is more logical to leave the existing property as as and add `get_formatted_device_time(self, format)` method, which is going to call the new POST. Also, in the documentation for this method it is necessary to mention it is only available since Appium 1.11

Reply:
@KazuCocoa @mykola-mokhnach 
Thanks a lot, you're right. I should have considered more.
I'll update along to your comments.